apisix/secret.lua

```diff
@@ -136,7 +136,7 @@ local function parse_secret_uri(secret_uri)
 end
 
 
-local function fetch_by_uri(secret_uri)
+local function fetch_by_uri_secret(secret_uri)
```

Comment:
Is this change necessary?

Reply:
I am using a new local function with same name here https://github.com/apache/apisix/pull/12682/files#diff-50c5241a560560f8d61637b60caa3e9f2dce27139863265cfa8ffb63a8537803R212 so this function name got changed to be more specific to avoid conflict.